<launch>
    <node pkg="loam_livox" type="get_feature"       name="get_feature"       output="screen" >
    <!--get feature param-->
    <param name="the_corner_min_curvature"          type="double"   value="0.02" />
  	<param name="the_surface_max_curvature"         type="double"   value="0.0005" />
  	<param name="the_min_hight"     		            type="double"   value="0.06" />
    <param name="view_max_distance"     		        type="double"   value="50" />
    <param name="view_min_distance"     		        type="double"   value="0.2" />
    <param name="outlier_min_distance"     		      type="double"   value="0.05" />
    <param name="the_min_intensity_difference"      type="double"   value="50" />
    <param name="the_max_intensity_hight"           type="double"   value="1.0" />
    <param name="the_min_slope_difference"          type="double"   value="0.3" />
    <param name="livox_z"     		                  type="double"   value="1.9921" />
    <param name="allow_get_feature"     		        type="bool"     value="true" />
    </node>
</launch>
